extensions/blocks/image-compare/view.scss

```diff
@@ -165,22 +165,20 @@ div.jx-image {
 	top: auto;
 }
 
-div.jx-image img {
-	height: 100%;
-	width: auto;
+div.jx-slider div.jx-image img {
+	height: 100% !important;
+	width: auto !important;
 	z-index: 5;
 	position: absolute;
 	margin-bottom: 0;
 
-	max-height: none;
-	max-width: none;
-	max-height: initial;
```

Comment:
I could swear there was a reason for the usage of `initial` here, (and the `none` preceeding it for back compat) but I can't recall what it is, and I couldn't find any issues with this change.